scripts/fake-ksops.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# Set up a fake ksops plugin
+
+tmpdir=$(mktemp -d $PWD/pluginsXXXXXX)
+trap "rm -rf $tmpdir" EXIT
+
+mkdir -p $tmpdir/viaduct.ai/v1/ksops
+ln -s /bin/true $tmpdir/viaduct.ai/v1/ksops/ksops
```

Comment:
We're creating a symlink from `$tmpdir/viaduct.ai/v1/ksops/ksops` to `/bin/true`. So when `kustomize` looks for the `ksops` plugin, it finds `$tmpdir/viaduct.ai/v1/ksops/ksops`, which is executable (because it's really `/bin/true`), it runs it, and `/bin/true`: (a) produces no output and (b) returns successfully, so everybody is happy.

Reply:
Thanks, that clears it up :)